crates/node/src/node.rs

```diff
@@ -230,21 +254,36 @@ where
     ) -> eyre::Result<NetworkHandle<OpNetworkPrimitives>> {
         let mut network_config = self.inner.network_config(ctx)?;
         // this is rolled with limited trusted peers and we want ignore any reputation slashing
+        
+        // Reset reputation weights for trusted peers
         network_config.peers_config.reputation_weights = ReputationChangeWeights::zero();
-        network_config.peers_config.backoff_durations.low = Duration::from_secs(5);
-        network_config.peers_config.backoff_durations.medium = Duration::from_secs(5);
-        network_config.peers_config.backoff_durations.high = Duration::from_secs(5);
+        
+        // Configure backoff settings
+        network_config.peers_config.backoff_durations.low = DEFAULT_BACKOFF_DURATION;
+        network_config.peers_config.backoff_durations.medium = DEFAULT_BACKOFF_DURATION;
+        network_config.peers_config.backoff_durations.high = DEFAULT_BACKOFF_DURATION;
         network_config.peers_config.max_backoff_count = u8::MAX;
-        network_config.sessions_config.session_command_buffer = 750;
-        network_config.sessions_config.session_event_buffer = 750;
+        
+        // Configure session buffers
+        network_config.sessions_config.session_command_buffer = DEFAULT_SESSION_BUFFER_SIZE;
+        network_config.sessions_config.session_event_buffer = DEFAULT_SESSION_BUFFER_SIZE;
 
+        // Configure transaction propagation
         let txconfig = TransactionsManagerConfig {
             propagation_mode: TransactionPropagationMode::All,
             ..network_config.transactions_manager_config.clone()
         };
+
+        // Initialize and start network
         let network = NetworkManager::builder(network_config).await?;
         let handle = ctx.start_network_with(network, pool, txconfig);
-        info!(target: "reth::cli", enode=%handle.local_node_record(), "P2P networking initialized");
+        
+        info!(
+            target: "reth::cli",
+            enode = %handle.local_node_record(),
+            "P2P networking initialized successfully"
```

Comment:
if it's initialized then this implies success, there's no such thing as initialized unsuccesful 
```suggestion
            "P2P networking initialized"
```